lib/elixir/src/elixir_interpolation.erl

```diff
@@ -52,12 +52,21 @@ extract(Line, Column, Scope, true, [$#, ${ | Rest], Buffer, Output, Last) ->
   end;
 
 extract(Line, Column, Scope, Interpol, [$\\, Char | Rest], Buffer, Output, Last) ->
-  extract(Line, Column+2, Scope, Interpol, Rest, [Char, $\\ | Buffer], Output, Last);
+  extract(Line, Column + 2, Scope, Interpol, Rest, [Char, $\\ | Buffer], Output, Last);
 
 %% Catch all clause
 
 extract(Line, Column, Scope, Interpol, [Char | Rest], Buffer, Output, Last) ->
-  extract(Line, Column + 1, Scope, Interpol, Rest, [Char | Buffer], Output, Last).
+  NewRest =
+    case Rest of
+      [$\r, $\n | _T] ->
```

Comment:
I had to add this validation, because when `unicode_util:gc/1` finds these tokens together it clusterizes them, and it was breaking the heredocs validation in the extract functions above